src/plugins/dashboard/public/dashboard_container/component/grid/dashboard_grid.tsx

```diff
@@ -34,14 +38,17 @@ export const DashboardGrid = ({
   viewportWidth: number;
 }) => {
   const dashboardApi = useDashboardApi();
+  const dashboardInternalApi = useDashboardInternalApi();
 
-  const [animatePanelTransforms, expandedPanelId, focusedPanelId, panels, useMargins, viewMode] =
+  const animatePanelTransforms = useStateFromPublishingSubject(
+    dashboardInternalApi.animatePanelTransforms$
+  );
+  const [expandedPanelId, focusedPanelId, panels, useMargins, viewMode] =
     useBatchedPublishingSubjects(
-      dashboardApi.animatePanelTransforms$,
       dashboardApi.expandedPanelId,
       dashboardApi.focusedPanelId$,
       dashboardApi.panels$,
-      dashboardApi.useMargins$,
+      dashboardApi.settings.useMargins$,
       dashboardApi.viewMode
     );
```

Comment:
Why not include `animatePanelTransforms` in the batched call?

Reply:
I am not sure. I moved it into the batched call 98f5bcf123a3e62e7d94ebbc36ad61d2fa03a79a